cms/static/js/views/bulkupdate.js

```diff
@@ -0,0 +1,341 @@
+                    }.bind(this));
+                }
+            },
+
+            /**
+             * Resets the BulkUpdate internally and visually
+             *
+             */
+            reset: function () {
+                current.stage = STAGE.SUBMITTING;
+                current.state = STATE.READY;
+
+                clearTimeout(timeout.id);
+                clearUpdate();
+                updateFeedbackList();
+                hideFeedbackList();
+            },
+
+            /**
+             * Show last update status from server and start sending requests
+             * to the server for status updates
+             *
+             * @return {jQuery promise}
+             */
+            resume: function () {
+                deferred = $.Deferred();
+                update = this.storedUpdate().update;
+
+                $.getJSON(update.statusUrl, function (data) {
+                    current.stage = data.UpdateStatus;
+                    if (current.stage < STAGE.SUBMITTING) {
+                        current.state = STATE.ERROR;
+                    } else if (current.stage === STAGE.UPDATING_PROBLEMS) {
+                        current.state = STATE.SUCCESS;
+                    } else if (current.stage === STAGE.VALIDATING ||
+                               current.stage === STAGE.UPDATING_ADVANCED_SETTINGS) {
+                        current.state = STATE.IN_PROGRESS;
+                    }
+                    displayFeedbackList();
+                    this.pollStatus(current.stage);
+                }.bind(this));
+
+                return deferred.promise();
+            },
+
+            /**
+             * Starts the updating process.
+             * Makes status list visible and starts showing update progress.
+             *
+             * @param {int} maxAttempts The new maxAttempt setting
+             * @param {string} showAnswer The new showAnswer setting
+             * @param {string} statusUrl The full URL to use to query the server
+             *     about the update status
+             * @return {jQuery promise}
+             */
+            start: function (maxAttempts, showAnswer, statusUrl) {
+                current.state = STATE.IN_PROGRESS;
+                deferred = $.Deferred();
+
+                update.maxAttempts = maxAttempts;
+                update.showAnswer = showAnswer;
+                update.statusUrl = statusUrl;
+
+                initEventListeners();
+                storeUpdate();
+                displayFeedbackList();
+                updateFeedbackList();
+
+                return deferred.promise();
+            },
+
+            /**
+             * Fetches the previous stored update
+             *
+             * @return {JSON} the data of the previous update
+             */
+            storedUpdate: function () {
+                return JSON.parse($.cookie(COOKIE_NAME));
+            }
+        };
+
+        return BulkUpdate;
+    });
```

Comment:
Can we drop the closing `};` down to a new line,
so that it lines up with the `define(`?

At first it looked like it wasn't being closed.

Reply:
done